.history/popup_20230518181610.css

```diff
@@ -0,0 +1,168 @@
+.no-scroll {
+  overflow: hidden;
+}
+
+/* wrapper */
+.ven-overlay {
+  position: fixed;
+  overflow-y: scroll;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 77px 0;
+  background: rgba(193, 199, 208, 0.4);
+}
+
+[aria-hidden="true"] {
+  display: none;
+}
+
+[aria-hidden="false"] {
+  display: block;
+}
+
+/* modal pop-up inner div */
+.ven-overlay .ven-modal-ul {
+  width: 80%;
+  margin: auto;
+  padding: 24px;
+  background: white;
+  border-radius: 16px;
+}
+
+.ven-modal-ul-li {
+  display: flex;
+  flex-direction: column;
+  gap: 36px;
+}
+
+.ven-title-canopy-cancel-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.ven-title-canopy-cancel-wrapper .ven-cancel-icon {
+  width: 12px;
+  height: 12px;
+}
+
+.ven-title-canopy-year {
+  display: flex;
+  flex-direction: column;
+}
+
+.ven-title-canopy-year h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+li {
+  list-style-type: none;
+}
+
+.ven-tech-used-stack li {
+  background: rgba(235, 235, 255, 1);
+  padding: 10px 12px;
+  border-radius: 8px;
+  text-align: center;
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.ven-canopy-list-wrapper li {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.ven-canopy-list-wrapper li:nth-child(3),
+.ven-canopy-list-wrapper li:nth-child(5) {
+  color: rgba(122, 134, 154, 1);
+}
+
+.ven-featured-img-wrapper img {
+  width: 100%;
+}
+
+.ven-tech-used-stack {
+  display: flex;
+  gap: 5px;
+}
+
+.ven-canopy-list-wrapper {
+  display: flex;
+  gap: 12px;
+}
+
+.ven-card-empty-list {
+  width: 8px;
+  height: 8px;
+  align-self: center;
+  background-color: #c1c7d0;
+  border-radius: 8px;
+}
+
+.ven-detail-stack-wrapper p,
+.ven-title-canopy-year h2 {
+  margin: 0;
+}
+
+.ven-detail-stack-wrapper {
+  display: flex;
+  gap: 36px;
+  margin-bottom: 111px;
+}
+
+.ven-tech-used {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.ven-sources {
+  display: flex;
+  justify-content: end;
+  gap: 15px;
+}
+
+.ven-sources button {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #6070ff;
+  padding: 4px 12px;
+  background: transparent;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  cursor: pointer;
+}
+
+.ven-sources button img {
+  align-self: last baseline;
+}
+
+.bg-blur {
+  filter: blur(19px);
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/* .hide{
+  display: none;
+} */
```

Comment:
- [ ] [OPTIONAL] For a clean code base, would you consider removing this unused and commented piece of code? This make your work more professional, avoid potential bugs, and make it more readable. :thinking: